
variant_vacio={
    "findings":
    [
        ['Variant Name', 'Gene', 'Variant Effect', 'Genotype', 'Result'],
        ('', '', '', '', '')
    ],
    "Summary":
    [
        ''
    ],

    "Other":
    [
        ''
    ],
    "Interpretations":
    [
        ["ClinVar","LOVD"],
        ("", "")
    ],
    
    "Population":
    [
        ["1000 Genomes","ESP65000","gnomAD"],
        ("","","")
    ],

    "inSilico" :
    [
        [17,1,7],
        [
            [' SIFT ', 'PolyPhen-2 HDIV', 'PolyPhen-2 HVAR', 'LRT', 'Mutation Taster', 'Mutation Assessor', 'Provean', 'MetaSVM', 'MetaLR', 'MetaRNN', 'M-CAP', 'MutPred Score', 'MPC Score', 
             'MVP Score', 'PrimateAI', 'DEOGEN2', 'BayesDel AddAF', 'BayesDel noAddAF', 'ClinPred', 'LIST-S2', 'Aloft', 'DANN Score', 'Fathm', 'Fathm MKL', 'Fathm MXF'],
             ('', '', '',"" ,'', '',"","","","","","","","","",
              "","","","","","","","","","")
        ]
    ],

    "Conservation":
    [
        ['PhyloP 1000W Vertebrate','PhyloP 30W Mammalian','PhastCons 100W Vertebrate','PhastCons 30W Mammalian','GERP++RS'],
        ('', '', '', '', '')
    ],
    "References":[['PubMed ID', 'Title', 'Year'],('', '', '',)],
    "type":""#Significant,Uncertain,Other
}

variant1={
    "findings":[
    ['Variant Name', 'Gene', 'Variant Effect', 'Genotype', 'Result'],
    ('p.S358L', 'TMEM43', 'SNV', 'Heterozygote', 'Pathogenic')
    ],
    
    "Summary":
    [
        'Arrhythmogenic right ventricular cardiomyopathy (Unknown)',
        'Arrhythmogenic right ventricular cardiomyopathy, type 5 (Autosomal dominant inheritance)',
        'Emery-Dreifuss muscular dystrophy 7, autosomal dominant (Autosomal dominant inheritance)' 
    ],

    "Other":
    [
        'TMEM43:NM_024334:exon12:c.C1073T:p.S358L'
    ],
    "Interpretations":
    [
        ["ClinVar","LOVD"],
        ("Pathogenic for Primary dilated cardiomyopathy, Cardiomyopathy, Hypertrophic cardiomyopathy, Arrhythmogenic right ventricular cardiomyopathy type 5, Familial isolated arrhythmogenic right ventricular dysplasia, Arrhythmogenic right ventricular cardiomyopathy, Cardiovascular phenotype",
         "pathogenic|VUS")
    ],
    
    "Population":
    [
        ["1000 Genomes","ESP65000","gnomAD"],
        ("","","")
    ],

    "inSilico" :
    [
        [17,1,7],
        [
            [' SIFT ', 'PolyPhen-2 HDIV', 'PolyPhen-2 HVAR', 'LRT', 'Mutation Taster', 'Mutation Assessor', 'Provean', 'MetaSVM', 'MetaLR', 'MetaRNN', 'M-CAP', 'MutPred Score', 'MPC Score', 
             'MVP Score', 'PrimateAI', 'DEOGEN2', 'BayesDel AddAF', 'BayesDel noAddAF', 'ClinPred', 'LIST-S2', 'Aloft', 'DANN Score', 'Fathm', 'Fathm MKL', 'Fathm MXF'],
             ('Deletereous', 'Damaging', 'Damaging',"Deletereous" ,'Known disease causing', 'Moderate',"Deletereous","Tolerated","Tolerated","Deletereous","Deletereous","0.891","0.526","0.380","Tolerated",
              "Tolerated","Deletereous","Deletereous","Deletereous","Deletereous",".","0.999","Tolerated","Deletereous","Deletereous")
        ]
    ],

    "Conservation":
    [
        ['PhyloP 1000W Vertebrate','PhyloP 30W Mammalian','PhastCons 100W Vertebrate','PhastCons 30W Mammalian','GERP++RS'],
        ('7.285', '0.976', '1.000', '0.978', '5.71')
    ],
    "References":[['PubMed ID', 'Title', 'Year'],('', '', '',)],
    "type":"Significant"
}

variant2={
    "findings":
    [
        ['Variant Name', 'Gene', 'Variant Effect', 'Genotype', 'Result'],
        ('p.A283D(!)', 'LAMA4', 'nonframeshift substitution', 'Homozygote', '')
    ],
    "Summary":
    [
        'Dilated cardiomyopathy 1JJ (Autosomal dominant inheritance)'
    ],

    "Other":
    [
        'LAMA4:NM_001105206:exon8:c.848_849delinsAC:p.A283D',
        'LAMA4:NM_001105207:exon8:c.827_828delinsAC:p.A276D',
        'LAMA4:NM_002290:exon8:c.827_828delinsAC:p.A276D'
    ],
    "Interpretations":
    [
        ["ClinVar","LOVD"],
        ("", "")
    ],
    
    "Population":
    [
        ["1000 Genomes","ESP65000","gnomAD"],
        ("","","")
    ],

    "inSilico" :
    [
        ['','',''],
        [
            [' SIFT ', 'PolyPhen-2 HDIV', 'PolyPhen-2 HVAR', 'LRT', 'Mutation Taster', 'Mutation Assessor', 'Provean', 'MetaSVM', 'MetaLR', 'MetaRNN', 'M-CAP', 'MutPred Score', 'MPC Score', 
             'MVP Score', 'PrimateAI', 'DEOGEN2', 'BayesDel AddAF', 'BayesDel noAddAF', 'ClinPred', 'LIST-S2', 'Aloft', 'DANN Score', 'Fathm', 'Fathm MKL', 'Fathm MXF'],
             ('', '', '',"" ,'', '',"","","","","","","","","",
              "","","","","","","","","","")
        ]
    ],

    "Conservation":
    [
        ['PhyloP 1000W Vertebrate','PhyloP 30W Mammalian','PhastCons 100W Vertebrate','PhastCons 30W Mammalian','GERP++RS'],
        ('', '', '', '', '')
    ],
    "References":[['PubMed ID', 'Title', 'Year'],('', '', '',)],
    "type":"Significant"
}

variant3={
    "findings":
    [
        ['Variant Name', 'Gene', 'Variant Effect', 'Genotype', 'Result'],
        ('p.I16555V(!) ', 'TTN', 'SNV', 'Heterozygote', 'VUS')
    ],
    "Summary":
    [
        'Primary dilated cardiomyopathy (Unknown)',
        'Dilated cardiomyopathy 1G (Autosomal dominant inheritance)',
        'Familial hypertrophic cardiomyopathy 9 (Autosomal dominant inheritance)',
        'Limb-girdle muscular dystrophy, type 2J (Autosomal recessive inheritance)',
        'Myopathy, early-onset, with fatal cardiomyopathy (Autosomal recessive inheritance)',
        'Myopathy, myofibrillar, 9, with early respiratory failure (Autosomal dominant inheritance)',
        'Tibial muscular dystrophy (Autosomal dominant inheritance)',
        'Microcephaly, short stature, and polymicrogyria with or without seizures (Autosomal recessive inheritance)'
    ],

    "Other":
    [
        'TTN:NM_003319:exon154:c.A49663G:p.I16555V',
        'TTN:NM_133432:exon155:c.A50038G:p.I16680V',
        'TTN:NM_133437:exon155:c.A50239G:p.I16747V',
        'TTN:NM_133378:exon275:c.A69154G:p.I23052V',
        'TTN:NM_001256850:exon276:c.A71935G:p.I23979V',
        'TTN:NM_001267550:exon326:c.A76858G:p.I25620V'
    ],

    "Interpretations":
    [
        ["ClinVar","LOVD"],
        ("Uncertain significance for Dilated cardiomyopathy 1G, Limb-girdle muscular dystrophy type 2J",
        "VUS")
    ],
    
    "Population":
    [
        ["1000 Genomes","ESP65000","gnomAD"],
        ("","","")
    ],

    "inSilico" :
    [
        [3,4,18],
        [
            [' SIFT ', 'PolyPhen-2 HDIV', 'PolyPhen-2 HVAR', 'LRT', 'Mutation Taster', 'Mutation Assessor', 'Provean', 'MetaSVM', 'MetaLR', 'MetaRNN', 'M-CAP', 'MutPred Score', 'MPC Score', 
             'MVP Score', 'PrimateAI', 'DEOGEN2', 'BayesDel AddAF', 'BayesDel noAddAF', 'ClinPred', 'LIST-S2', 'Aloft', 'DANN Score', 'Fathm', 'Fathm MKL', 'Fathm MXF'],
             ('Tolerated', 'Benign', 'Benign','' ,'Disease causing', 'Neutral ','Neutral','Tolerated','Tolerated' ,'Tolerated','Tolerated','','0.356 ','0.088','Tolerated',
              '','Tolerated ','Tolerated','Tolerated','Tolerated','','0.870','Tolerated','Deletereous','Deletereous')
        ]
    ],

    "Conservation":
    [
        ['PhyloP 1000W Vertebrate','PhyloP 30W Mammalian','PhastCons 100W Vertebrate','PhastCons 30W Mammalian','GERP++RS'],
        ('5.945 ', '1.138', '1.000 ', '0.999', '4.76')
    ],
    "References":[['PubMed ID', 'Title', 'Year'],('', '', '',)],
    "type":"uncertain"
}
variant4={
    "findings":
    [
        ['Variant Name', 'Gene', 'Variant Effect', 'Genotype', 'Result'],
        ('p.A13755P(!)', 'TTN', 'SNV', 'Heterozygote', 'VUS'),
    ],
    "Summary":
    [
       ' Primary dilated cardiomyopathy (Unknown)',
        'Dilated cardiomyopathy 1G (Autosomal dominant inheritance)',
        'Familial hypertrophic cardiomyopathy 9 (Autosomal dominant inheritance)',
        'Limb-girdle muscular dystrophy, type 2J (Autosomal recessive inheritance)',
        'Myopathy, early-onset, with fatal cardiomyopathy (Autosomal recessive inheritance)',
        'Myopathy, myofibrillar, 9, with early respiratory failure (Autosomal dominant inheritance)',
        'Tibial muscular dystrophy (Autosomal dominant inheritance)',
        'Microcephaly, short stature, and polymicrogyria with or without seizures (Autosomal recessive inheritance)'
    ],

    "Other":
    [
        'TTN:NM_003319:exon150:c.G41263C:p.A13755P',
        'TTN:NM_133432:exon151:c.G41638C:p.A13880P',
        'TTN:NM_133437:exon151:c.G41839C:p.A13947P',
        'TTN:NM_133378:exon271:c.G60754C:p.A20252P',
        'TTN:NM_001256850:exon272:c.G63535C:p.A21179P',
        'TTN:NM_001267550:exon322:c.G68458C:p.A22820P'

    ],

    "Interpretations":
    [
        ["ClinVar","LOVD"],
        ("Conflicting interpretations of pathogenicity for Cardiomyopathy, Hypertrophic cardiomyopathy, Tibial muscular dystrophy, Myopathy myofibrillar 9 with early respiratory failure, Dilated cardiomyopathy 1G, Limb-girdle muscular dystrophy type 2J, Myopathy early-onset with fatal cardiomyopathy, Familial hypertrophic cardiomyopathy 9, not specified, Cardiovascular phenotype",
        "VUS|benign|likely benign")
    ],
    
    "Population":
    [
        ["1000 Genomes","ESP65000","gnomAD"],
        ("0.000798722 ","0.0022 ","0.0012")
    ],

    "inSilico" :
    [
        [15,4,6],
        [
            [' SIFT ', 'PolyPhen-2 HDIV', 'PolyPhen-2 HVAR', 'LRT', 'Mutation Taster', 'Mutation Assessor', 'Provean', 'MetaSVM', 'MetaLR', 'MetaRNN', 'M-CAP', 'MutPred Score', 'MPC Score', 
             'MVP Score', 'PrimateAI', 'DEOGEN2', 'BayesDel AddAF', 'BayesDel noAddAF', 'ClinPred', 'LIST-S2', 'Aloft', 'DANN Score', 'Fathm', 'Fathm MKL', 'Fathm MXF'],
             ('Deletereous', 'Damaging', 'Damaging','' ,'Disease causing', 'High ','Deletereous','Deletereous','Deletereous' ,'Tolerated','Deletereous','','0.827 ','0.526','Deletereous',
              '','Tolerated ','Deletereous','Tolerated','Deletereous','','0.916 ','Tolerated','Deletereous','Deletereous')
        ]
    ],

    "Conservation":
    [
        ['PhyloP 1000W Vertebrate','PhyloP 30W Mammalian','PhastCons 100W Vertebrate','PhastCons 30W Mammalian','GERP++RS'],
        ('7.858', '1.026', '1.000 ', '1.000', '5.83')
    ],
    "References":[['PubMed ID', 'Title', 'Year'],('', '', '',)],
    "type":"Uncertain"
}
variant5={
    "findings":    
    [
        ['Variant Name', 'Gene', 'Variant Effect', 'Genotype', 'Result'],
        ('p.P11492T', 'TTN', 'SNV', 'Heterozygote', 'VUS'),
    ],
    
    "Summary":
    [
        'Primary dilated cardiomyopathy (Unknown)',
        'Dilated cardiomyopathy 1G (Autosomal dominant inheritance)',
        'Familial hypertrophic cardiomyopathy 9 (Autosomal dominant inheritance)',
        'Limb-girdle muscular dystrophy, type 2J (Autosomal recessive inheritance)',
        'Myopathy, early-onset, with fatal cardiomyopathy (Autosomal recessive inheritance)',
        'Myopathy, myofibrillar, 9, with early respiratory failure (Autosomal dominant inheritance)',
        'Tibial muscular dystrophy (Autosomal dominant inheritance)',
        'Microcephaly, short stature, and polymicrogyria with or without seizures (Autosomal recessive inheritance)'
    ],

    "Other":
    [
        'TTN:NM_001267550:exon149:c.C34474A:p.P11492T',
    ],

    "Interpretations":
    [
        ["ClinVar","LOVD"],
        ("Conflicting interpretations of pathogenicity for Dilated cardiomyopathy 1G, Limb-girdle muscular dystrophy type 2J, not specified",
         "benign|likely benign")
    ],
    
    "Population":
    [
        ["1000 Genomes","ESP65000","gnomAD"],
        ("0.00159744","","0.0028")
    ],

    "inSilico" :
    [
        [2,15,8],
        [
            [' SIFT ', 'PolyPhen-2 HDIV', 'PolyPhen-2 HVAR', 'LRT', 'Mutation Taster', 'Mutation Assessor', 'Provean', 'MetaSVM', 'MetaLR', 'MetaRNN', 'M-CAP', 'MutPred Score', 'MPC Score', 
             'MVP Score', 'PrimateAI', 'DEOGEN2', 'BayesDel AddAF', 'BayesDel noAddAF', 'ClinPred', 'LIST-S2', 'Aloft', 'DANN Score', 'Fathm', 'Fathm MKL', 'Fathm MXF'],
             ('', '', '','' ,'Disease causing ', '','','','' ,'Tolerated','','','0.742','','',
              '','Tolerated ','Tolerated','','Tolerated','','0.697 ','Tolerated','Deletereous','Neutral')
        ]
    ],

    "Conservation":
    [
        ['PhyloP 1000W Vertebrate','PhyloP 30W Mammalian','PhastCons 100W Vertebrate','PhastCons 30W Mammalian','GERP++RS'],
        ('0.672', '1.176', '0.935', '1.000', '5.95')
    ],
    "References":[['PubMed ID', 'Title', 'Year'],('', '', '',)],
    "type":"Uncertain"
}

variant6={
    "findings":
    [
        ['Variant Name', 'Gene', 'Variant Effect', 'Genotype', 'Result'],
        ('p.K1288R', 'DSP', 'SNV', 'Heterozygote', 'VUS'),
    ],
    "Summary":
    [
        'Arrhythmogenic right ventricular cardiomyopathy (Unknown)',
        'Arrhythmogenic right ventricular dysplasia 8 (Heterogeneous|Autosomal dominant inheritance)',
        'Cardiomyopathy, dilated, with woolly hair, keratoderma, and tooth agenesis (Autosomal dominant inheritance)',
        'Dilated cardiomyopathy with woolly hair and keratoderma (Autosomal recessive inheritance)',
        'Keratosis palmoplantaris striata II (Autosomal dominant inheritance)',
        'Lethal acantholytic epidermolysis bullosa (Autosomal recessive inheritance)',
        'Skin fragility-woolly hair-palmoplantar keratoderma syndrome (Autosomal recessive inheritance)',
        'Deafness, autosomal dominant 39, with dentinogenesis imperfecta 1 (Autosomal dominant inheritance)',
        'Denticles (Autosomal dominant inheritance)',
        'Dentinogenesis imperfecta type 2 (Autosomal dominant inheritance)',
        'Dentinogenesis imperfecta type 3 (Autosomal dominant inheritance)'

    ],

    "Other":
    [
        'DSP:NM_001319034:exon23:c.A3863G:p.K1288R',
        'DSP:NM_004415:exon23:c.A3863G:p.K1288R'

    ],

    "Interpretations":
    [
        ["ClinVar","LOVD"],
        ("Uncertain significance for Cardiomyopathy",
         "likely benign|VUS")
    ],
    
    "Population":
    [
        ["1000 Genomes","ESP65000","gnomAD"],
        ("","","3.187e-05")
    ],

    "inSilico" :
    [
        [12,1,12],
        [
            [' SIFT ', 'PolyPhen-2 HDIV', 'PolyPhen-2 HVAR', 'LRT', 'Mutation Taster', 'Mutation Assessor', 'Provean', 'MetaSVM', 'MetaLR', 'MetaRNN', 'M-CAP', 'MutPred Score', 'MPC Score', 
             'MVP Score', 'PrimateAI', 'DEOGEN2', 'BayesDel AddAF', 'BayesDel noAddAF', 'ClinPred', 'LIST-S2', 'Aloft', 'DANN Score', 'Fathm', 'Fathm MKL', 'Fathm MXF'],
             ('Tolerated', 'Damaging', 'Damaging','Deletereous' ,'Disease causing', 'Low','Neutral','Deletereous','Deletereous' ,'Tolerated','Deletereous','0.101','0.812','0.182 ','Tolerated',
              'Tolerated','Tolerated','Tolerated','Tolerated','Tolerated','','0.997 ','Deletereous','Deletereous','Deletereous')
        ]
    ],

    "Conservation":
    [
        ['PhyloP 1000W Vertebrate','PhyloP 30W Mammalian','PhastCons 100W Vertebrate','PhastCons 30W Mammalian','GERP++RS'],
        ('2.138', '1.312', '1.000 ', '0.998', '5.17')
    ],
    "References":[['PubMed ID', 'Title', 'Year'],('', '', '',)],
    "type":"Uncertain"
}
variant7={
    "findings":
    [
        ['Variant Name', 'Gene', 'Variant Effect', 'Genotype', 'Result'],
        ('p.T449I(!)', 'CACNB2', 'SNV', 'Heterozygote', 'VUS'),
    ],
    "Summary":
    [
        'Brugada syndrome 4 (Autosomal dominant inheritance)'

    ],

    "Other":
    [
        'CACNB2:NM_000724:exon13:c.C1346T:p.T449I',
        'CACNB2:NM_001330060:exon13:c.C1232T:p.T411I',
        'CACNB2:NM_201570:exon13:c.C1367T:p.T456I',
        'CACNB2:NM_201590:exon13:c.C1349T:p.T450I',
        'CACNB2:NM_001167945:exon14:c.C1313T:p.T438I',
        'CACNB2:NM_201571:exon14:c.C1427T:p.T476I',
        'CACNB2:NM_201572:exon14:c.C1355T:p.T452I',
        'CACNB2:NM_201593:exon14:c.C1397T:p.T466I',
        'CACNB2:NM_201596:exon14:c.C1511T:p.T504I',
        'CACNB2:NM_201597:exon14:c.C1439T:p.T480I'

    ],

    "Interpretations":
    [
        ["ClinVar","LOVD"],
        ("Conflicting interpretations of pathogenicity for Ventricular tachycardia, Brugada syndrome 4, Brugada syndrome, not specified, Cardiovascular phenotype",
         "benign|VUS|likely benign")
    ],
    
    "Population":
    [
        ["1000 Genomes","ESP65000","gnomAD"],
        ("0.000399361","0.0015","0.0015")
    ],

    "inSilico" :
    [
        [7,2,16],
        [
            [' SIFT ', 'PolyPhen-2 HDIV', 'PolyPhen-2 HVAR', 'LRT', 'Mutation Taster', 'Mutation Assessor', 'Provean', 'MetaSVM', 'MetaLR', 'MetaRNN', 'M-CAP', 'MutPred Score', 'MPC Score', 
             'MVP Score', 'PrimateAI', 'DEOGEN2', 'BayesDel AddAF', 'BayesDel noAddAF', 'ClinPred', 'LIST-S2', 'Aloft', 'DANN Score', 'Fathm', 'Fathm MKL', 'Fathm MXF'],
             ('Tolerated', 'Possibly damaging', 'Benign','Neutral' ,'Probably damaging', 'Low','Neutral','Tolerated','Tolerated' ,'Tolerated','Deletereous','','0.878','0.219','Tolerated',
              'Tolerated','Tolerated','Tolerated','Tolerated','Deletereous','','0.998','Deletereous','Deletereous','Neutral')
        ]
    ],

    "Conservation":
    [
        ['PhyloP 1000W Vertebrate','PhyloP 30W Mammalian','PhastCons 100W Vertebrate','PhastCons 30W Mammalian','GERP++RS'],
        ('0.904', '0.947', '0.987', '0.997', '3.56')
    ],
    "References":[['PubMed ID', 'Title', 'Year'],('', '', '',)],
    "type":"Uncertain"
}
variant8={
        "findings":
    [
        ['Variant Name', 'Gene', 'Variant Effect', 'Genotype', 'Result'],
        ('g.55665584A>C', 'TNNI3', '', 'Homozygote', 'VUS'),
    ],
    "Summary":
    [
        'Dilated cardiomyopathy 1FF (Unknown)',
        'Dilated cardiomyopathy 2A (Autosomal recessive inheritance)',
        'Familial hypertrophic cardiomyopathy 7 (Heterogeneous|Autosomal dominant inheritance)',
        'Familial restrictive cardiomyopathy 1 (Autosomal dominant inheritance)',
        'Cardiac conduction disease with or without dilated cardiomyopathy (Autosomal dominant inheritance)'

    ],

    "Other":
    [
        'NC_000019.9:g.55665584A>C'
    ],

    "Interpretations":
    [
        ["ClinVar","LOVD"],
        ("Conflicting interpretations of pathogenicity for Cardiomyopathy, Hypertrophic cardiomyopathy, Primary ciliary dyskinesia, Familial restrictive cardiomyopathy 1, Dilated cardiomyopathy 2A, Familial hypertrophic cardiomyopathy 7, Primary familial hypertrophic cardiomyopathy, not specified, Familial Hypertrophic Cardiomyopathy with Wolff-Parkinson-White Syndrome, Nemaline Myopathy Recessive",
         "benign|likely benign")
    ],
    
    "Population":
    [
        ["1000 Genomes","ESP65000","gnomAD"],
        ("1","","1")
    ],

    "inSilico" :
    [
        [7,2,16],
        [
            [' SIFT ', 'PolyPhen-2 HDIV', 'PolyPhen-2 HVAR', 'LRT', 'Mutation Taster', 'Mutation Assessor', 'Provean', 'MetaSVM', 'MetaLR', 'MetaRNN', 'M-CAP', 'MutPred Score', 'MPC Score', 
             'MVP Score', 'PrimateAI', 'DEOGEN2', 'BayesDel AddAF', 'BayesDel noAddAF', 'ClinPred', 'LIST-S2', 'Aloft', 'DANN Score', 'Fathm', 'Fathm MKL', 'Fathm MXF'],
             ('', '', '','' ,'', '','','' ,'','','','','','',
              '','','','','','','','','','')
        ]
    ],

    "Conservation":
    [
        ['PhyloP 1000W Vertebrate','PhyloP 30W Mammalian','PhastCons 100W Vertebrate','PhastCons 30W Mammalian','GERP++RS'],
        ('', '', '', '', '')
    ],
    "References":[['PubMed ID', 'Title', 'Year'],('', '', '',)],
    "type":"Uncertain"
}

datos={
    "variant":{"p.S358L":variant1,"p.A283D(!)":variant2,"p.I16555V(!)":variant3,"p.A13755P(!)":variant4,"p.P11492T":variant5,"p.K1288R":variant6,"p.T449I(!)":variant7,"g.55665584A>C":variant8},

    "info":[
        {"gen":"TITIN (TTN)","desc":"This gene encodes a large abundant protein of striated muscle. The product of this gene is divided into two regions, a N-terminal I-band and a C-terminal A-band. The I-band, which is the elastic part of the molecule, contains two regions of tandem immunoglobulin domains on either side of a PEVK region that is rich in proline, glutamate, valine and lysine. The A-band, which is thought to act as a protein-ruler, contains a mixture of immunoglobulin and fibronectin repeats, and possesses kinase activity. An N-terminal Z-disc region and a C-terminal M-line region bind to the Z-line and M-line of the sarcomere, respectively, so that a single titin molecule spans half the length of a sarcomere. Titin also contains binding sites for muscle associated proteins so it serves as an adhesion template for the assembly of contractile machinery in muscle cells. It has also been identified as a structural protein for chromosomes. Alternative splicing of this gene results in multiple transcript variants. Considerable variability exists in the I-band, the M-line and the Z-disc regions of titin. Variability in the I-band region contributes to the differences in elasticity of different titin isoforms and, therefore, to the differences in elasticity of different muscle types. Mutations in this gene are associated with familial hypertrophic cardiomyopathy 9, and autoantibodies to titin are produced in patients with the autoimmune disease scleroderma. [provided by RefSeq, Feb 2012]"
        },
        {"gen":"TRANSMEMBRANE PROTEIN 43 (TMEM43)","desc" : "This gene belongs to the TMEM43 family.Defects in this gene are the cause of familial arrhythmogenic right ventricular dysplasia type 5 (ARVD5), also known as arrhythmogenic right ventricular cardiomyopathy type 5 (ARVC5). Arrhythmogenic right ventricular dysplasia is an inherited disorder, often involving both ventricles, and is characterized by ventricular tachycardia, heart failure, sudden cardiac death, and fibrofatty replacement of cardiomyocytes. This gene contains a response element for PPAR gamma (an adipogenic transcription factor), which may explain the fibrofatty replacement of the myocardium, a characteristic pathological finding in ARVC. [provided by RefSeq, Oct 2008]"
        },
        {"gen":"DESMOPLAKIN (DSP)","desc" : "This gene encodes a protein that anchors intermediate filaments to desmosomal plaques and forms an obligate component of functional desmosomes. Mutations in this gene are the cause of several cardiomyopathies and keratodermas, including skin fragility-woolly hair syndrome. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Jan 2016]"
        },

        {"gen":"CALCIUM VOLTAGE-GATED CHANNEL AUXILIARY SUBUNIT BETA 2 (CACNB2)","desc" :"This gene encodes a subunit of a voltagedependent calcium channel protein that is a member of the voltage-gated calcium channel superfamily. The gene product was originally identified as an antigen target in Lambert-Eaton myasthenic syndrome, an autoimmune disorder. Mutations in this gene are associated with Brugada syndrome. Alternatively spliced variants encoding different isoforms have been described. [provided by RefSeq, Feb 2013]"
        },
        {"gen":"TROPONIN I3, CARDIAC TYPE (TNNI3)" ,"desc" : "Troponin I (TnI), along with troponin T (TnT) and troponin C (TnC), is one of 3 subunits that form the troponin complex of the thin filaments of striated muscle. TnI is the inhibitory subunit; blocking actin-myosin interactions and thereby mediating striated muscle relaxation. The TnI subfamily contains three genes: TnI-skeletal-fast-twitch, TnI-skeletal-slow-twitch, and TnI-cardiac. This gene encodes the TnI-cardiac protein and is exclusively expressed in cardiac muscle tissues. Mutations in this gene cause familial hypertrophic cardiomyopathy type 7 (CMH7) and familial restrictive cardiomyopathy (RCM). Troponin I is useful in making a diagnosis of heart failure, and of ischemic heart disease. An elevated level of troponin is also now used as indicator of acute myocardial injury in patients hospitalized with moderate/severe Coronavirus Disease 2019 (COVID-19). Such elevation has also been associated with higher risk of mortality in cardiovascular disease patients hospitalized due to COVID-19. [provided by RefSeq, Aug 2020]"
        },
        {"gen":"LAMININ SUBUNIT ALPHA 4 (LAMA4)","desc" :"Laminins, a family of extracellular matrix glycoproteins, are the major noncollagenous constituent of basement membranes. They have been implicated in a wide variety of biological processes including cell adhesion, differentiation, migration, signaling, neurite outgrowth and metastasis. Laminins are composed of 3 non identical chains: laminin alpha, beta and gamma (formerly A, B1, and B2, respectively) and they form a cruciform structure consisting of 3 short arms, each formed by a different chain, and a long arm composed of all 3 chains. Each laminin chain is a multidomain protein encoded by a distinct gene. Several isoforms of each chain have been described. Different alpha, beta and gamma chain isomers combine to give rise to different heterotrimeric laminin isoforms which are designated by Arabic numerals in the order of their discovery, i.e. alpha1beta1gamma1 heterotrimer is laminin 1. The biological functions of the different chains and trimer molecules are largely unknown, but some of the chains have been shown to differ with respect to their tissue distribution,presumably reflecting diverse functions in vivo. This gene encodes the alpha chain isoform laminin, alpha 4. The domain structure of alpha 4 is similar to that of alpha 3, both of which resemble truncated versions of alpha 1 and alpha 2, in that approximately 1,200 residues at the N-terminus (domains IV, V and VI) have been lost. Laminin, alpha 4 contains the C-terminal G domain which distinguishes all alpha chains from the beta and gamma chains. The RNA analysis from adult and fetal tissues revealed developmental regulation of expression, however, the exact function of laminin, alpha 4 is not known. Tissue-specific utilization of alternative polyA-signal has been described in literature. Alternative splicing results in multiple transcript variants encoding distinct isoforms. [provided by RefSeq, Aug 2011]"
        },
    ]
    ,
    "anexo":{"Header":"ANNEX I. GENERAL CONSIDERATIONS AND LIMITATIONS",
            "Genome":"GRCh37 (hg19)",
            "Sequencer": "Ion GeneStudio S5 System",
            "Analysis":"Ion Torrent Suite, Variant Caller v5.16-6 (2ccc1088), Ogmios Platform",
            "Sample": ["Fixation:","Concentration:","Volume:","Concentration:","Celularity: "],
            "Database": ["ClinVar (https://www.ncbi.nlm.nih.gov/clinvar/)",
                            "LOVD (https://www.lovd.nl/)",
                            "1000 Genomes (https://www.internationalgenome.org/)",
                            "ESP6500 (https://evs.gs.washington.edu/EVS/)",
                            "gnomAD (https://gnomad.broadinstitute.org/)",
                            "NCBI Gene (https://www.ncbi.nlm.nih.gov/gene)",
                            "NCBI Medgen (https://www.ncbi.nlm.nih.gov/medgen/)",
                            "dbnsfp (https://sites.google.com/site/jpopgen/dbNSFP)",
                            "dbscsnv (http://www.liulab.science/dbscsnv.html)",
                            "UCSC Common (https://genome.ucsc.edu/)",
                            "Variomes (https://candy.text-analytics.ch/Variomes/)"
                            ]
    }
}

Sample={
    "SampleID":"XXXX",
    "Petitioner":"XXXX",
    "Hospital":"XXXX",
    "SampleDate":"XXXX",
    "BIONOSDate":"XXXX",
    "UPVDate":"XXXX",
    "ReportDate":"XXXX",
    "updateDate":"XXXX",
    "Diagnostic/reason":"XXXX",
    "Comments":"XXXX"
}


